### ISSUE TITLE
Content place field not stored

### ISSUE THREAD

**Issue by ralphboeije** [part-up/part-up#53]
When starting a part-up and changing the image several time in start details, the place field valued is not stored and set to "null" (e.g. https://pu-acceptance.lifely.nl/partups/tesate-FYWQ27tMrgqHk8mkm)

![schermafbeelding 2015-12-18 om 13 51 53](https://cloud.githubusercontent.com/assets/14044235/11897276/a742cae4-a58e-11e5-81ee-a5617c533daa.png)

![location and budget settings bug](https://cloud.githubusercontent.com/assets/14044235/11897279/b01b204e-a58e-11e5-984b-86d9a1938876.png)

**Comment by jessedvrs** [comment]
You mean in this screen right? This is what I tried to reproduce in Chrome:

1. Log in into Part-up
2. Click Start a part-up
3. Continue to /start/details with the orange button 
4. Scroll down to the location field and fill in & select Amsterdam, Netherlands
5. Scroll further down to the image field and upload an image. Then re-upload an image. Then re-re-upload an image. Then re-re-re-upload an image. Then re-re-re-re-upload an image.

The location field still says "Amsterdam, Netherlands".
If the steps to reproduce are wrong, could you please provide the correct steps?

<img width="821" alt="screen shot 2016-01-21 at 16 25 16" src="https://cloud.githubusercontent.com/assets/5961598/12484662/9985702c-c05b-11e5-80a6-f9878faecd63.png">

**Comment by ralphboeije** [comment]
Did you also create the part-up? It left location null only when you created the part-up for real. But it is really strange, I just retested and can't reproduce it anymore, although I retested it back then time and time again. 

There is however something wrong with this place field. If you have a part-up where the end date is in the past and you change it to a date in the future, it sets the date value to undefined.

<img width="1440" alt="screen shot 2016-01-21 at 17 17 33" src="https://cloud.githubusercontent.com/assets/14044235/12486527/67a93266-c063-11e5-9d36-5391c4cd5684.png">
